docs/src/main/asciidoc/kafka.adoc

```diff
@@ -2054,6 +2057,76 @@ class BaristaTest {
 The application will process this message and send a message to `beverages` channel.
 <5> Use the `received` method on `beverages` channel to check the messages produced by the application.
 
+If your Kafka consumer is batch based, you will need to send a batch of messages to the channel as by creating them manually.
+
+For instance:
+
+[source, java]
+----
+@ApplicationScoped
+public class BeverageProcessor {
+
+    @Incoming("orders")
+    @Outgoing("beverages")
```

Comment:
The processing method doesn't return any outgoing data. You need to remove this `Outgoing` annotation and the beverages sink in the test code.

Reply:
updated